watertap/costing/units/gac.py

```diff
@@ -22,142 +22,99 @@
 
 
 class ContactorType(StrEnum):
-    pressure = "pressure"
-    gravity = "gravity"
+    pressure = "pres"
+    gravity = "grav"
```

Comment:
Why make this change? I think it was better before ...

Reply:
I'd prefer to change it back too for the sake of clarity.